build.gradle

```diff
@@ -10,8 +10,7 @@ plugins {
 
 repositories {
     mavenLocal()
-    jcenter()
-
+    mavenCentral()
```

Comment:
The `mavenCentral()` entry is better off coming first to work around accidental local builds getting in the way.  Next time you're in here maybe just move it up.